src/main/types.ts

```diff
@@ -1,10 +1,18 @@
-export type ToggleMap = Map<string, IToggleDescription>
+export type ToggleMap = Map<string, IToggleDetails>
 
-export interface IToggleDescription {
-    id: string
+export interface IToggleDetails {
+    type: string
     fraction: number
     description?: string
     comment?: string
 }
 
+export interface IToggle {
+    [key: string]: IToggleDetails
```

Comment:
Curious about this. I am guessing that this was changed so a toggle could be looked up by key. Is using the current structure, an array of toggles possible? If the type were added to the original IToggleDescription and made optional would it be possible to not be a breaking change?